src/api/controllers/lines.js

```diff
@@ -342,16 +342,27 @@ function updateLineStatus(req, res) {
                 });
             } else {
                 try {
-                    var msg = await line.updateStatus(req.body.status);
-                    res.status(200).json({
-                          "message": "Successfully updated line with status " + req.body.status
-                    });
-                } catch (err) {
-                    console.log(err);
-                    res.status(400).json({
-                        "message": err
-                    });
-                }
+                    if (req.body.status === 'startrecruiter' || req.body.status === 'finishrecruiter' && (user.user_type !== 'recruiter' || user.user_type !== 'admin')) {
+                        res.status(401).json({
+                            "message": response.unauthorized
```

Comment:
create a new message in response.js (use command + T in atom and type that) 
{
...
recruitersOrAdmins: "UnauthorizedError: Only available to recruiters or admins"
...
}